js/models.js

```diff
@@ -0,0 +1,32 @@
+App.Models.Todo = Backbone.Model.extend({
+    defaults: {
+        'isComplete': false,
+    },
+
+    toggleComplete: function () {
+        this.set({ 'isComplete': !this.get('isComplete') });
+        this.save();
+    },
+
+    // Use LocalStorage
+    sync: function (method, model, options = {}) {
+        let key = 'Todo-' + model.cid;
+
+        if (method === 'create' || method === 'update') {
+            localStorage.setItem(key, JSON.stringify(model));
+        }
+        else if (method === 'read') {
+            let result = localStorage.getItem(key);
+            if (result) {
+                result = JSON.parse(result);
+                options.success && options.success(result);
+            }
+            else if (options.error) {
+                options.error("Todo " + model.cid + " not found.");
+            }
+        }        
+        else if (method === 'delete') {
+            localStorage.removeItem(key);
+         }
+    }
```

Comment:
I'm not 100% sure on this, but I think I would be tempted to do storage at the Collection or the App level. Just because at some point when you're booting the app, you can just pull all the keys out of storage and use them to build out the collection of models. I think breaking them down into individual keys is the right thing to do because of the 10MB limit, but I'd be temped to just listen to changes in the models and handle saving at the Collection level. I could be convinced that this is a bad idea though :P

Also, does destroy() call sync() by default? I think you might need to do that yourself by overriding `destroy()`?